contracts/voting_escrow/src/contract.rs

```diff
@@ -786,14 +792,36 @@ fn withdraw_early_callback(
     let return_astro_amount = current_astro_balance.saturating_sub(precallback_astro);
 
     if !return_astro_amount.is_zero() {
-        let transfer_msg = SubMsg::new(WasmMsg::Execute {
-            contract_addr: config.astro_addr.to_string(),
-            msg: to_binary(&Cw20ExecuteMsg::Transfer {
-                recipient: slashed_funds_receiver.to_string(),
-                amount: return_astro_amount,
-            })?,
-            funds: vec![],
-        });
+        // Check that slashed_funds_receiver is an escrow_fee_distributor contract
+        // otherwise transfer return_astro_amount to slashed_funds_receiver address
+        let version: StdResult<ContractVersion> =
+            deps.querier.query(&QueryRequest::Wasm(WasmQuery::Raw {
+                contract_addr: slashed_funds_receiver.to_string(),
+                key: CONTRACT.as_slice().into(),
+            }));
+        let transfer_msg = match version {
+            Ok(ContractVersion { contract, .. })
+                if contract.eq("astroport-escrow_fee_distributor") =>
```

Comment:
There are only dashes, no underscores in the contract name

Reply:
Changed underscores to dashes in fee distributor CONTRACT_NAME, thanks